app/views/admin/configs/show.html.erb

```diff
@@ -1328,4 +1366,6 @@
       </div>
     </div>
   </div>
+  <%#= render partial: "modal" %>
```

Comment:
Suggestion: remove this comment. There's not actually a partial being rendered here: a dom node is being inserted by javascript!

Reply:
Oops! Will do!

Reply:
@joshpuetz done!